src/features/auth/pages/signup/AcceptCgu/AcceptCgu.tsx

```diff
@@ -104,62 +106,64 @@ export const AcceptCgu: FC<PreValidationSignupStepProps> = ({ signUp }) => {
           isVisible={isDoingReCaptchaChallenge}
         />
       )}
-      <CardContent>
-        <Paragraphe>
-          <Typo.Body>En cliquant sur “Accepter et s’inscrire”, tu acceptes nos </Typo.Body>
-          <ExternalLink text="Conditions Générales d’Utilisation" url={env.CGU_LINK} primary />
-          <Spacer.Row numberOfSpaces={1} />
-          <Typo.Body> ainsi que notre </Typo.Body>
-          <ExternalLink
-            text="Politique de confidentialité."
-            primary
-            url={env.PRIVACY_POLICY_LINK}
-          />
-        </Paragraphe>
-        <Spacer.Column numberOfSpaces={5} />
-        <Paragraphe>
-          <Typo.Body>
-            Pour en savoir plus sur la gestion de tes données personnelles et exercer tes droits tu
-            peux&nbsp;:
-          </Typo.Body>
-        </Paragraphe>
-        <ExternalTouchableLink
-          as={ButtonTertiaryPrimary}
-          wording="Contacter le support"
-          accessibilityLabel="Ouvrir le gestionnaire mail pour contacter le support"
-          externalNav={contactSupport.forGenericQuestion}
-          icon={Email}
-        />
-        <Spacer.Column numberOfSpaces={6} />
-        <ButtonPrimary
-          wording="Accepter et s’inscrire"
-          accessibilityLabel="Accepter les conditions générales d’utilisation et la politique de confidentialité pour s’inscrire"
-          // Token needs to be a non-empty string even when ReCaptcha validation is deactivated
-          // Cf. backend logic for token validation
-          onPress={onSubmit}
-          isLoading={isDoingReCaptchaChallenge || isFetching}
-          disabled={disabled}
-          accessibilityDescribedBy={checkCGUErrorId}
-        />
-        <InputError
-          visible={!!errorMessage}
-          messageId={errorMessage}
-          numberOfSpacesTop={5}
-          relatedInputId={checkCGUErrorId}
-        />
-        <Spacer.Column numberOfSpaces={5} />
-      </CardContent>
+      <StyledTitle3>CGU & Données</StyledTitle3>
```

Comment:
Si tu as envie tu peux faire ça, parce que c'est pas vraiment du style
```suggestion
      <Typo.Title3 {...getHeadingAttrs(2)}>CGU & Données</Typo.Title3>
```